Submitty/Submitty
# [VPAT: Discussion Forum] Fixes W3 Validator A not allowed in B errors

### Please check if the PR fulfills these requirements:


### What is the current behavior?
Addresses #5131 

### What is the new behavior?
Fixes validator errors for A not allowed in B.

### Other information?
Tested using w3 validator (how to test): 
1. Open forum
2. Save page as html
3. Upload html to https://validator.w3.org/#validate_by_upload
4. See error

Features affected by this PR:
- Links in thread list
- Thread list scrolling
- Reply to all and unresolve button
- Buttons for sorting by Hierarchical, Chronological, Alpha
- Filter Buttons for threads

Note: If you view the forum as an instructor you may get a warning that looks like
`Attribute xlink not allowed on element svg at this point`
This is HTML for the `flatpickr` dependency that we use.
